### PR TITLE
feat: Tes auth

### PR DESCRIPTION
**Pull request by vschnei**
### QC

* [x] The PR contains a test case for the changes or the changes are already covered by an existing test case.
* [x] The documentation (`docs/`) is updated to reflect the changes or this is not necessary (e.g. if the change does neither modify the language nor the behavior or functionalities of Snakemake).

### CODE REVIEW

File: docs/executing/cloud.rst [hunk 1]
```diff
@@ -438,3 +441,16 @@ you can forward your token by setting the `TES_TOKEN` environmental variable.
...
     $ export TES_TOKEN=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c
+
+**Funnel basic authentication:** 
+In order to execute individual tasks for Funnel based TES servers, a basic authentication is required .
+An authentication via AOuth2 access token is not supported yet.
+
+.. code-block:: console
+
+    $ snakemake \
+        --tes $TES_URL --tes_user $FUNNEL_USER --tes_password $FUNNEL_PASSWORD\
```
Comment (johanneskoester) [inline docs/executing/cloud.rst:452]: I would not pass credentials like this (cumbersome cause it needs to be repeated, can lead to leaking passwords via bash history). Rather, the credentials should be given as env vars.
Suggestion: `TES_USER` and `TES_PASSWORD`. Or maybe funnel already has some expected env vars for this to be used?

**Review by johanneskoester — CHANGES_REQUESTED** [review]
Thanks! One thing below

File: docs/executing/cloud.rst (hunk 1 above)
Comment (vschnei) [reply docs/executing/cloud.rst]: That make sense.

Nobody should be able to see the password!

**Review by johanneskoester — APPROVED** [review]

**Comment by johanneskoester** [comment]
Ci issues are fixed, but the TES tests fail now.

**Pull request merged**